ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -295,6 +295,9 @@ aaa_authorization:
   serial_console: < true | false >
   commands:
     all_default: < group group_name | local | none > < group group_name | local | none >
+    privilege:
```

Comment:
This must be a list as I am able to pass multiple levels and group value combinations.

privileges:
  - level: < privilege level(s) >
    group: < group_name(s) | tacacs+ local >

Reply:
Please check this once as I am not sure completely.